e2e/utils/css-utils.ts

```diff
@@ -1,6 +1,10 @@
 import {Page} from 'puppeteer';
 
-export async function clickByXpath(page: Page, xpathSelector: string) {
+/**
+ * Summary: Helper functions when directly dealing with CSS selectors.
+ */
+
+export async function clickByEvalXpath(page: Page, xpathSelector: string) {
```

Comment:
Ideally we could have an abstraction where you just call `click()` and we pick the strategy behind the scenes.